certora/specs/math.spec

```diff
@@ -0,0 +1,79 @@
+
+methods {
+    min(uint256, uint256)           returns (uint256)       envfree
+    max(uint256, uint256)           returns (uint256)       envfree
+    zeroFloorSub(uint256, uint256)  returns (uint256)       envfree
+    divUp(uint256, uint256)         returns (uint256)       envfree
+}
+
+/// min ///
+
+rule minSafety(uint256 a, uint256 b) {
```

Comment:
Rules are fixed when run with the flag ` --settings -useBitVectorTheory`

Reply:
oh amazing